### ISSUE TITLE
Muscle ultrasound hyperechogenicity

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7163]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
619461   PMID: 32776697  1/7

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Definition: An increased echo intensity of muscle tissue on sonography, defined as an increased amount of returning echoes per square area of muscle tissue. On cross-section, normal muscle appears as a relatively anechoic structure with hyperechoic speckles within the tissue representing perimysial septa, giving it a starry night appearance. The boundaries of each muscle are delineated by the presence of hyperechoic fascia. In muscle disorders, one of the hallmark findings is replacement of healthy muscle with fat and fibrosis, manifested by an increase in echogenicity from higher sound transitions in the muscle. This increase in echogenicity is most distinct in conditions that lead to chronic pathology, such as long-standing muscle inflammation, dystrophy, or denervation.
PMID: 32989482

**Comment by pnrobinson** [comment]
added
